roles/pf9-hostagent/files/pf9_get_hostagent.py.j2

```diff
@@ -238,6 +246,11 @@ def main():
         action="store",
         help="Installer platform. Allowed options are redhat or debian.",
         type='choice', choices=['redhat', 'debian'])
+    parser.add_option(
+        '--proxy',
+        dest="proxy",
+        action="store",
+        default="", help="Proxy to reach internet")
```

Comment:
You may want to call this out aa <proxy addr>:<port> without prepending http/https in the help string, otherwise the split won't give you expected results.